hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/RBFConfigKeys.java

```diff
@@ -279,6 +279,10 @@ public class RBFConfigKeys extends CommonConfigurationKeysPublic {
       FEDERATION_ROUTER_PREFIX + "safemode.expiration";
   public static final long DFS_ROUTER_SAFEMODE_EXPIRATION_DEFAULT =
       3 * DFS_ROUTER_CACHE_TIME_TO_LIVE_MS_DEFAULT;
+  public static final String DFS_ROUTER_SAFEMODE_CHECKPERIOD =
```

Comment:
We are indirectly reducing the safe mode period but what we are really doing is adding a separate argument for this.
I agree that this should happen.
Let's just change the JIRA and PR title to "RBF: Add a safe mode check period configuration".

Reply:
Sure. Thank you

Reply:
Should we write the unit after the variable? DFS_ROUTER_SAFEMODE_CHECKPERIOD_S?

Reply:
Thanks for review.  Test case is in testRouterExitSafemode.

Reply:
@LiuGuH Thanks for the contribution! Thanks @goiri  for helping with the review.

It would be better if the variable name could indicate the time unit.

Reply:
@slfan1989 Thanks for your advise. Add MS for variable.